### DISCUSSION TITLE
Forgive me, controlling away from home?

### DISCUSSION THREAD

**Discussion by roastyduck** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#795] [Q&A]
Is this the mqtt? 

Spent a while trying to work out how to use it but it fried my brain and I got nowhere.. is there an easy how to guide?

**Comment by visualapproach** [comment]
MQTT talks to a 'broker'. Think of it as a server. To control it from away from home you need a secure way to talk to the broker.
Best way is to use a smart home like home assistant which comes with a broker and secure connection to the internet.

**Reply by roastyduck** [reply]
That makes sense, i don't suppose you have a link to a good easy guide or video if you may? I feel bad asking as you've done so much already with this.

Thank you

**Reply by visualapproach** [reply]
No but google has a lot of stuff ;)
